releases/release-4.0-ga.md

```diff
@@ -0,0 +1,114 @@
+    - Fix system panics caused by incorrect processing logic when creating the table that is `key` partitioned [#17242](https://github.com/pingcap/tidb/pull/17242)
+    - Fix the issue that the wrong `Index Merge Join` plan is selected because of incorrect optimizer processing logic [#17365](https://github.com/pingcap/tidb/pull/17365)
+    - Fix the issue of inaccurate `duration` monitoring metric of the `SELECT` statement in Grafana [#16561](https://github.com/pingcap/tidb/pull/16561)
+    - Fix the issue that the GC worker is blocked when the system error occurs [#16915](https://github.com/pingcap/tidb/pull/16915)
+    - Fix the issue that the `UNIQUE` constraint on a boolean column results in an incorrect result in a comparison [#17306](https://github.com/pingcap/tidb/pull/17306)
+    - Fix system panics caused by incorrect processing logic when `tidb_opt_agg_push_down` is enabled and the aggregation function pushes down the partitioned table [#17328](https://github.com/pingcap/tidb/pull/17328)
+    - Fix the issue of accessing failed TiKV nodes in some cases [#17342](https://github.com/pingcap/tidb/pull/17342)
+    - Fix the issue that the `isolation-read` configuration item in `tidb.toml` does not take effect [#17322](https://github.com/pingcap/tidb/pull/17322)
+    - Fix the issue of incorrect order of output results due to incorrect processing logic when `hint` is used to enforce the stream aggregation [#17347](https://github.com/pingcap/tidb/pull/17347)
+    - Fix the behavior that `insert` processes DIV under different `SQL_MODE` [#17314](https://github.com/pingcap/tidb/pull/17314)
+
+* TiFlash
+    - Fix the issue that the matching behavior of regular expressions in the  search log feature is inconsistent with other components
+    - Fix the issue of excessive restart time when nodes write large amounts of data by disabling the delay processing optimization of `Raft Compact Log Command` by default
+    - Fix the issue that the system fails to start because TiDB incorrectly processes the `DROP DATABASE` statement in some scenarios
+    - Fix the issue that the method of collecting CPU information in `Server_info` is different from that in other components
+    - Fix the issue that the error `Too Many Pings` is reported when the `Query` statement is executed if `batch coprocessor` is enabled
+    - Fix the issue that Dashboard fails to display the correct `deploy path` information because TiFlash does not report the related information
+
+* TiKV
+    - Fix the `DefaultNotFound` error that occurs when backing up using BR  [#7937](https://github.com/tikv/tikv/pull/7937)
+    - Fix system panics caused by out-of-order `ReadIndex` packets [#7930](https://github.com/tikv/tikv/pull/7930)
+    - Fix the unexpected error is returned because the read request callback function is not called [#7921](https://github.com/tikv/tikv/pull/7921)
```

Comment:
```suggestion
    - Fix the issue that an unexpected error is returned because the read request callback function is not called [#7921](https://github.com/tikv/tikv/pull/7921)
```